web/src/features/hfiCalculator/components/FuelTypeDropdown.tsx

```diff
@@ -0,0 +1,57 @@
+import makeStyles from '@mui/styles/makeStyles'
+import { TextField, Autocomplete } from '@mui/material'
+import { FuelType, WeatherStation } from 'api/hfiCalculatorAPI'
+import { StationInfo } from 'features/hfiCalculator/slices/hfiCalculatorSlice'
+import { isEqual, isNull } from 'lodash'
+import React from 'react'
+
+export interface FuelTypeDropDownProps {
+  station: WeatherStation
+  stationInfo?: StationInfo
+  fuelTypes: FuelType[]
+  setFuelType: (code: number, fuelTypeId: number) => void
+}
+
+const useStyles = makeStyles({
+  dropdownClass: {
+    width: 120
+  }
+})
+
+const FuelTypeDropDown = ({
```

Comment:
```suggestion
const FuelTypeDropdown = ({
```

Reply:
pedantic I know, but I foresee myself getting confused/frustrated later since the filename uses "Dropdown" (consistent with how we've cased other dropdown components)